### PR TITLE
[PAR-WD-FT] - Nina . [ongoing]

### PR DESCRIPTION
**Pull request by Gorlanova**

### CODE REVIEW

File: starter-code/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "starter-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon app.js -e .hbs,.js"
```
Comment (ta-web-paris) [inline starter-code/package.json]: `nodemon app.js -e .hbs,.js`
Should be
`nodemon app.js -e hbs,js`
https://www.npmjs.com/package/nodemon/v/1.19.2#specifying-extension-watch-list

File: starter-code/views/gallery.hbs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <div class="grid">
+            <div class="cell"><img src="/images/1473f5f8b4d2b444dfa4866d0f5b19ca.jpg" alt=""></div>
+            <div class="cell"><img src="/images/61CngygatqL._SL1500_.jpg" alt=""></div>
+            <div class="cell"><img src="/images/blooming-persian-lilac_53876-86166.jpg" alt=""></div>
+            <div class="cell"><img src="/images/maxresdefault.jpg" alt=""></div>
+            <div class="cell"><img src="/images/Michael-Scott-Misquotations-Office-Video.jpg" alt=""></div>
+            <div class="cell"><img src="/images/Products62075-1200x1200-1649647.jpg" alt=""></div>
+            <div class="cell"><img src="/images/SayNoToCilantro_400x400.png" alt=""></div>
+            <div class="cell"><img src="/images/this_is_fine.jpg" alt=""></div>
+            <div class="cell"><img src="/images/unnamed.jpg" alt=""></div>
```
Comment (ta-web-paris) [inline starter-code/views/gallery.hbs]: Here you could have put the images in an array and used the {{#each}} to display them : p

**Comment by ta-web-paris** [comment]
Well done ! I liked the design : )

**Comment by Gorlanova** [comment]
Thanks !! I’ll keep updating it so I’ll prob use an array and try a dynamic display in the gallery :)

**Pull request closed without merging**